crates/iota/src/validator_commands.rs

```diff
@@ -539,22 +539,20 @@ impl IotaValidatorCommand {
                     print_unsigned_transaction_only,
                 )?;
 
-                // in the Move code (`bridge::committee::register`) we use
-                // `active_validator_addresses` to get the list of active validators,
-                // so here we allow all active validators to register as members of the
-                // `BridgeCommittee`.
+                // The bridge should be run by the same committee as the consensus, hence se use
```

Comment:
`hence se use` => `hence we use`

Reply:
Thanks. Fixed it at [1bd676e](https://github.com/iotaledger/iota/pull/6015/commits/1bd676eefa541d047905a7b01729e857d25a1d95).